src/Modal/Modal.js

```diff
@@ -197,14 +232,13 @@ Modal.Header = ({ title, children, showClose = true }) => {
 Modal.Body = createComponent({
   name: 'ModalBody',
   style: css`
-    padding: 1.25rem;
+    padding-bottom: 24px;
   `,
 });
 
 Modal.Footer = createComponent({
   name: 'ModalFooter',
   style: ({ theme }) => css`
-    padding: 1rem 1.25rem;
     background: ${theme.colors.grayLightest};
```

Comment:
Curious why we have this `background` here...Was this intentional? 

<img width="354" alt="Screen Shot 2019-08-06 at 12 50 36 PM" src="https://user-images.githubusercontent.com/6404663/62572557-df98cd00-b848-11e9-8b53-153aac99cf09.png">

Reply:
Ha. Removing.